src/schema/v2/order/sharedTypes.ts

```diff
@@ -0,0 +1,449 @@
+  GraphQLEnumType,
+  GraphQLInt,
+  GraphQLList,
+  GraphQLNonNull,
+  GraphQLObjectType,
+  GraphQLString,
+  GraphQLUnionType,
+} from "graphql"
+import { ResolverContext } from "types/graphql"
+import { InternalIDFields } from "../object_identification"
+import { Money, resolveMinorAndCurrencyFieldsToMoney } from "../fields/money"
+import { ArtworkVersionType } from "../artwork_version"
+import { ArtworkType } from "../artwork"
+
+/**
+ * The order json as received from the exchange REST API.
+ * Used to nudge our our OrderType resolvers
+ */
+interface OrderJSON {
+  id: string
+  code: string
+  source: "artwork_page" | "inquiry" | "private_sale" | "partner_offer"
+  mode: "buy" | "offer"
+  currency_code: string
+  available_shipping_countries: string[]
+  buyer_phone_number?: string
+  buyer_phone_number_country_code?: string
+  buyer_total_cents?: number
+  shipping_total_cents?: number
+  items_total_cents?: number
+  shipping_name?: string
+  shipping_address_line1?: string
+  shipping_address_line2?: string
+  shipping_city?: string
+  shipping_postal_code?: string
+  shipping_region?: string
+  shipping_country?: string
+  fulfillment_options: Array<{
+    type: string
+    amount_minor: number
+    selected?: boolean
+  }>
+  line_items: Array<{
+    id: string
+    artwork_id: string
+    artwork_version_id: string
+    edition_set_id?: string
+    list_price_cents: number
+    quantity: number
+    shipping_total_cents?: number
+    currency_code: string
+  }>
+}
+
+const OrderModeEnum = new GraphQLEnumType({
+  name: "OrderModeEnum",
+  values: {
+    BUY: {
+      value: "BUY",
+    },
+    OFFER: {
+      value: "OFFER",
+    },
+  },
+})
+
+const OrderSourceEnum = new GraphQLEnumType({
+  name: "OrderSourceEnum",
+  values: {
+    ARTWORK_PAGE: {
+      value: "ARTWORK_PAGE",
+    },
+    INQUIRY: {
+      value: "INQUIRY",
+    },
+    PRIVATE_SALE: {
+      value: "PRIVATE_SALE",
+    },
+    PARTNER_OFFER: {
+      value: "PARTNER_OFFER",
+    },
+  },
+})
+
+// Enum for fulfillment_option.type field
+const FulfillmentOptionTypeEnum = new GraphQLEnumType({
+  name: "FulfillmentOptionTypeEnum",
+  values: {
+    DOMESTIC_FLAT: { value: "DOMESTIC_FLAT" },
+    INTERNATIONAL_FLAT: { value: "INTERNATIONAL_FLAT" },
+    PICKUP: { value: "PICKUP" },
+    SHIPPING_TBD: { value: "SHIPPING_TBD" },
+  },
+})
+
+type FulfillmentOptionJSONWithCurrencyCode = OrderJSON["fulfillment_options"][number] & {
+  _currencyCode: string
+}
+const FulfillmentOptionType = new GraphQLObjectType<
+  FulfillmentOptionJSONWithCurrencyCode,
+  ResolverContext
+>({
+  name: "FulfillmentOption",
+  fields: {
+    type: {
+      type: new GraphQLNonNull(FulfillmentOptionTypeEnum),
+      resolve: ({ type }) => {
+        if (type === "domestic_flat") return "DOMESTIC_FLAT"
+        if (type === "international_flat") return "INTERNATIONAL_FLAT"
+        if (type === "pickup") return "PICKUP"
+        if (type === "shipping_tbd") return "SHIPPING_TBD"
```

Comment:
@erikdstock - mentioned this before, but do we need to do this remapping? Why not just use the values exactly as they're coming back from the backend?